File-Formats/XlsIO/Worksheet-to-Image-Conversion.md

```diff
@@ -15,26 +15,14 @@ Refer to the following links for assemblies/nuget packages required based on pla
 * [Assemblies Information](https://help.syncfusion.com/file-formats/xlsio/assemblies-required#converting-excel-worksheet-to-image) 
 * [NuGet Information](https://help.syncfusion.com/file-formats/xlsio/nuget-packages-required#converting-excel-worksheet-to-image)
 
+N> Worksheet To Image conversion can be performed by referring .NET Standard assemblies in UWP platform. 
```

Comment:
Worksheet To Image conversion can be performed by referring Syncfusion.XlsIORenderer.Net.Core Nuget package in UWP platform.